content/versions-comparison/2.2-news.json

```diff
@@ -1,20 +1,20 @@
 [
   {
-    "title":"Rules on product models 🤖",
+    "title":"Rules compatible with product models 🤖",
     "domains": ["Rules engine", "Productivity"],
     "editions": ["EE"],
     "since-version": "2.2",
     "link": "/pim/v2/articles/manage-your-rules.html#execute-rules"
   },
   {
-    "title":"Search in the proposal grid 🔍",
+    "title":"Filters added to the proposals grid 🔍",
```

Comment:
Idem here, it sounds vaguer. We added a search bar, it's not really filters in terms of UX.